Libplanet.Action/State/IBlockChainStates.cs

```diff
@@ -35,21 +35,38 @@ public interface IBlockChainStates
         /// </list>
         /// </exception>
         /// <seealso cref="IWorldState"/>
-        IWorldState GetWorldState(BlockHash? offset);
+        IWorldState GetWorldState(BlockHash offset);
```

Comment:
Removed nullable parameter, since it's meaningless.
We can get empty `IWorldState` by injecting `null` on `IWorldState GetWorldState(HashDigest<SHA256>? stateRootHash)`

Reply:
Is it ok on Lib9c?